.ci_support/osx_64_.yaml

```diff
@@ -1,13 +1,15 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.9'
+- '10.13'
+MACOSX_SDK_VERSION:
+- '10.13'
```

Comment:
This is just updating the macOS minimum to 10.13, which is inline with this work ( https://github.com/conda-forge/conda-forge.github.io/issues/1844 )